tests/loggers/test_progress_bar_logger.py

```diff
@@ -66,6 +69,91 @@ def get_mock_tqdm(bar_format: str, *args: object, **kwargs: object):
 
     trainer.fit()
 
+    if dist.get_local_rank() != 0:
```

Comment:
There seems to be some duplicative code between the two tests...curious would it be possible to refactor them to use shared helpers?